ts/components/screens/BaseScreenComponent.tsx

```diff
@@ -59,10 +66,24 @@ class BaseScreenComponent extends React.PureComponent<Props, State> {
   }
 
   private showHelp = () => {
+    maybeDark(this.props.dark).map(_ =>
+      setStatusBarColorAndBackground(
+        "light-content",
+        customVariables.brandDarkGray
```

Comment:
```suggestion
        customVariables.colorWhite
```
If it is dark we have to show the status bar white, is it right?

Reply:
Yes, on the copy and paste of new notation i forgot to change the text and background color, I have just committed the fix